Cross references to Insights

lamagnifica opened the pull request:
@WatsonEmily , @shnayder , @catong , @srpearce , @mhoeber : added cross references to Insights. (Spell check also revealed some problems including smart quotes and apostrophes.) Not expected to land for several more days, accompanies https://github.com/edx/edx-analytics-dashboard/pull/225.

mhoeber commented:
:+1:  thanks

shnayder commented on en_us/course_authors/source/creating_content/create_problem.rst:
```diff
@@ -281,6 +281,10 @@ ribbon at the top of the page.
 
+In addition, edX Insights uses the display name to identify each problem.
+Unique, descriptive display names help you identify problems quickly and
+accurately for analysis. For more information, see `Using edX Insights`_.
```
+1000 for encouraging better display names :)

shnayder commented on en_us/course_authors/source/creating_content/create_problem.rst:
```diff
@@ -293,8 +297,8 @@ changed to a specific number, the **Maximum Attempts** setting for individual
- higher are included on the Student Answer Distribution report that you can
- download during your course.
+ higher are included in the answer distribution computations used in edX
+ Insights and the Student Answer Distribution report.
```
does this mean that problems with unlimited attempts aren't included?? That would be surprising... 

If it just means that problems with max_attempts=0 (e.g. students can't answer them at all) aren't included, perhaps just say that.

lamagnifica replied:
@mulby  and @brianhw , what is your take on this? I believe this wording was crafted with your help back when the SAD report went out. 
@shnayder , I will restate as you suggest if it is a straightforward max_attempts = 0 limitation. But, don't you think that "1 or higher" would be understood to cover unlimited attempts?

shnayder replied:
I think you make it 'null' to mean unlimited, and I got confused. (I'm surprised you can even make a problem have 0 attempts--I would have guessed that 0 actually means unlimited).

mulby replied:
I can't think of any reason why it would have to have a maximum attempts setting, @brianhw do you remember?

lamagnifica replied:
@brianhw?

brianhw replied:
My memory is that this stemmed from the practice of some courses to use problems as a way to conduct polls.  Since the polls didn't have a "right" answer, there was no "check" button enabled, and the user would instead be asked to "save".  We wanted to make it clear that answers to such problems would not appear in any answer distribution we were generating, because the distribution was limited to problem_check events.  I don't remember the specific wording one way or the other, but that was the intention.

WatsonEmily commented:
:+1:

srpearce commented on en_us/course_authors/source/creating_content/create_problem.rst:
```diff
@@ -313,7 +317,7 @@ problem. The problem weight appears next to the problem title.
        points circled
 
-By default, each response field, or “answer space,” in a Problem
+By default, each response field, or "answer space, in a Problem
```
Missing closing quotation mark

srpearce commented on en_us/course_authors/source/running_course/course_answers.rst:
```diff
@@ -15,6 +15,9 @@ a selected problem.
 
 * :ref:`score_histogram`
 
+Student answer distribution data, including both charts and reports, are also
```
are -> is (refers to "data," not "charts and reports")

srpearce commented:
Two minor issues - otherwise :+1:

lamagnifica commented:
need to add change log?

lamagnifica commented:
Added similar changes to developing_course/course_units.rst, course_subsections.rst, and course_sections.rst. Corrected some problems in the course_units.rst also. @mhoeber

lamagnifica commented:
closed in favor of https://github.com/edx/edx-documentation/pull/199
